backend/tracim_backend/models/context_models.py

```diff
@@ -668,11 +668,11 @@ def is_deleted(self) -> bool:
         return self.workspace.is_deleted
 
     @property
-    def calendar_enabled(self) -> bool:
+    def agenda_enabled(self) -> bool:
```

Comment:
Please add an an explanation (a comment) about why the property is not renamed

Reply:
it is .